_pages/continuous-deployment.md

```diff
@@ -86,7 +86,8 @@ For an example manifest and manifest-staging see here:
 
 
 ## Zero Downtime Deploy Options
-- [`v3-zdt-push`](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html) is an official command, yet is in active development.
-- `zero-downtime-push` is the popular Autopilot plugin used by a lot of TTS projects and used in both of the above examples. It is now unmaintained and archived though. Does not support buildpacks. If your application successfully deploys to cloud.gov but does not start, which may happen for an application that does not have an adequate test suite, you may have to go into the cf target space and manually delete the "APP_NAME-venerable" application in order to make use of `autopilot` again.
+- {%include components/tag-standard.html %} The native Cloud Foundry [rolling app deployments](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html#deploy) CLI is preferred over other options.  
+- {%include components/tag-caution.html %} [Autopilot](https://github.com/contraband/autopilot) is a plugin historically used by a lot of TTS projects and used in both of the above examples. **It is now unmaintained and archived and does not support buildpacks.** We recommend moving to the official CF deployment commands. 
```

Comment:
it feels like we should add examples that are using the native commands, but that is out of scope for this issue.